cluster-provision/os-3.11/scripts/provision.sh

```diff
@@ -71,6 +73,10 @@ git clone https://github.com/openshift/openshift-ansible.git -b v3.11.0 --depth
 # TODO: remove it when the fix will be available under the v3.11.0 tag
 sed -i 's/python-docker/python-docker-py/' $openshift_ansible/playbooks/init/base_packages.yml
 
+# Apply fix https://github.com/openshift/openshift-ansible/issues/11213
+# TODO: remove it when the fix will be available under the v3.11.0 tag
```

Comment:
we execute `git clone https://github.com/openshift/openshift-ansible.git -b v3.11.0 --depth 1 $openshift_ansible`
tag v3.11.0 is quite old and because of that we have to apply bugfixes by ourselves, isn't better to switch to openshift-ansible-3.11.118-1 ([link](https://github.com/openshift/openshift-ansible/releases))?

Reply:
I'd say: go ahead and give it a try :)

Reply:
I thought that somebody may have a reason to use this specific tag, but I will give it a try.